docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/container-set-function.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
You need to set the link on new page into _data/sidebars/scos_dev_sidebar.yml

Reply:
++